tests/test_metadata.py

```diff
@@ -240,10 +244,12 @@ def test_general_labels(
         if container_type != ImageType.APPLICATION:
             assert "BCI" in labels[f"{prefix}.title"]
 
-        assert (
-            "based on the SLE Base Container Image."
-            in labels[f"{prefix}.description"]
-        )
+        # distribution images do not hold this label
```

Comment:
nitpick: this is not correct, the label exists, just does not contain the sentence "based on the SLE Base Container Image."

Reply:
amended. let's wait for the CI.